core/arch/aarch64/emit_utils.c

```diff
@@ -213,15 +214,15 @@ insert_exit_stub_other_flags(dcontext_t *dcontext, fragment_t *f, linkstub_t *l,
         /* The final slot is a data slot, which will hold the address of either
          * the fcache-return routine or the linked fragment.
          */
+        ASSERT(pc == target_pc_slot || pc + 1 == target_pc_slot);
```

Comment:
The +1 case is alignment padding?

Reply:
Yes.

Reply:
I added some clarification in the comment above.